### PR TITLE
feat(network): modify bitxhub network.toml config struct

### PR DESCRIPTION
**Pull request by treasersimplifies**
1. Transform struct of network.toml to support multiple addresses for one node (use the first address as default address).

### CODE REVIEW

File: cmd/bitxhub/key.go [hunk 1]
```diff
@@ -8,8 +8,6 @@ import (
 
 	"github.com/meshplus/bitxhub-kit/key"
```
Comment (sandyzhou) [inline cmd/bitxhub/key.go:9]: import order?

File: internal/repo/network.go [hunk 2]
```diff
@@ -64,10 +148,19 @@ func loadNetworkConfig(repoRoot string) (*NetworkConfig, error) {
 	m := make(map[uint64]*peer.AddrInfo)
 	for _, node := range nodes {
 		if node.ID != networkConfig.ID {
-			addr, err := AddrToPeerInfo(node.Addr)
+			// addr, err := AddrToPeerInfo(node.Addr)
```
Comment (sandyzhou) [inline internal/repo/network.go:151]: remove the redundant code

File: internal/repo/network.go [hunk 3]
```diff
@@ -76,3 +169,111 @@ func loadNetworkConfig(repoRoot string) (*NetworkConfig, error) {
...
+			panic(err)
+		}
+		fmt.Println(i, ":", id.String())
+		peerIDs = append(peerIDs, id.String())
+	}
+	return peerIDs
+}
+
+// PrintNetworkConfig .
+func PrintNetworkConfig(networkConfig *NetworkConfig) {
```
Comment (Alexader) [inline internal/repo/network.go:262]: unused code.

File: internal/repo/network.go [hunk 4]
```diff
@@ -2,24 +2,47 @@ package repo
...
 	"strings"
 
+	"github.com/libp2p/go-libp2p-core/crypto"
 	"github.com/libp2p/go-libp2p-core/peer"
+	"github.com/meshplus/bitxhub/pkg/cert"
 	ma "github.com/multiformats/go-multiaddr"
 )
 
+const (
+	networkConfigFile    = "network.toml"
```
Comment (Alexader) [inline internal/repo/network.go:17]: unused variable

**Pull request merged**